src/commands/tick-log.js

```diff
@@ -58,7 +62,14 @@ function log(argv) {
       break
   }
 }
-
+function writeFilterError() {
```

Comment:
@sofia-scho could you add a blank line above and below this function? We want to try and keep the code looking consistent with the other functions.